src/components/Dropdown/Dropdown.jsx

```diff
@@ -396,7 +399,8 @@ Dropdown.defaultProps = {
   readOnly: false,
   isLoading: false,
   loadingMessage: undefined,
-  ariaLabel: undefined
+  ariaLabel: undefined,
+  filterOption: undefined
```

Comment:
It's already `undefined` by default, no?

Reply:
Yeah, should be. I guess just for explicitly...
We'll refactor when we do TS migration